webapp/client/src/components/FormFieldRadioGroup.js

```diff
@@ -19,7 +19,6 @@ const Radio = styled.div`
   }
 
   input[type="radio"] + label {
-    white-space: pre-wrap;
```

Comment:
You added this line with #516. I guess that it was copied to here. I am not so sure why it was added (that was the reason for my question) but I am wondering if we can just delete it. I guess it was added to fulfill some design criteria as this [deals with new lines](https://developer.mozilla.org/en-US/docs/Web/CSS/white-space)

Reply:
This was added because there was a strange behavior in the radio field when using a Trans object. This was fixed in a different branch and can be deleted.